carbon-design-system/carbon
# fix(UIShell): expand check so side nav items close properly

Closes #9725 

`displayName` wasn't cutting it when we check for side nav items to pass the `isSideNavExpanded` attribute onto. So this PR expands that check to also look for just a reagular ol function name 👍🏾 



#### Testing / Reviewing

- go to the rail mode side nav in the UIShell
- unfurl a side nav list item
- hover off the side nav causing it to close
- observe the list ite REFURL 🏄🏾
